index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
+            </div>
+            <div class="tonic-icons-container">
+                <ul>
+                    <li><img src="images/html-tag.png" alt="html-tag"></li>
+                    <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                    <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+               </ul>
+            </div>
+            <div class="tonic-btn">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+            <div class="tonic-container">
+                <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
+                <div class="tonic-heading">Multi-Post Stories</div>
+                <div class="canopy-container">
+                    <div class="canopy-header">CANOPY</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">Back End Dev</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">2015</div>
+                </div>
+                <div class="tonic-parag">
+                    <p>
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="tonic-icons-container">
+                    <ul>
+                        <li><img src="images/html-tag.png" alt="html-tag"></li>
+                        <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                        <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+                   </ul>
+                </div>
+                <div class="tonic-btn">
+                    <button type="button"> See Project</button>
+              </div>
+            </div>
+                
+</section>
+<footer class="footer-container">
+    <h1>
+        About <br>
+        Myself
+    </h1>
+    <p>
+        I'm a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.  
+    </p>
+<div class="social-media">
+<h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
+    <ul>
+        <li><a href="#"><img src="images/twitter-icon.png" alt="twitter-icon"></a></li> 
+        <li><a href="#"><img src="images/linkedin-icon.png" alt="linkedin-icon"></a></li> 
+        <li><a href="#"><img src="images/vector-one.png" alt="vector-one"></a></li>
+        <li><a href="#"><img src="images/github-icon.png" alt="github-icon"></a></li> 
+        <li><a href="#"><img src="images/vector-two.png" alt="vector-two"></a></li> 
+    </ul>
+</div>
+<div class="tonic-btn">
+    <button type="button"> Get my resume</button>
+</div>
```

Comment:
- [ ] Please follow the button styling: Active, Hover, and Pressed for the **"Get my resume"** button. :+1: 

![b1](https://user-images.githubusercontent.com/70483835/164852523-763d4ee4-690a-4bb0-af24-a233be199117.png)